canary/apps/react/vite/package.json

```diff
@@ -5,26 +5,19 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "postbuild": "yarn run size-limit",
     "preview": "vite preview",
     "start": "vite preview --port 3000"
   },
   "dependencies": {
     "@aws-amplify/ui-react": "latest",
     "aws-amplify": "latest",
-    "react": "latest",
-    "react-dom": "latest"
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1"
```

Comment:
Is there a pattern as to why we're moving `react` / `react-dom` away from `latest` while we're moving `vue` / `vite` to `latest`?

Reply:
Not really, this just aligns more closely with how the generated package.json from the vite cli looks. This change is pretty temporary and we'll be removing the canary folder altogether when we merge v1 and v2 build tests